service/manager.go

```diff
@@ -45,6 +42,10 @@ type ExitCallback func(shutdown Exit) int
 // Implements Manager interface
 type manager struct {
 	serviceCore
+
+	StartTimeout time.Duration `yaml:"start_timeout" default:"10s"`
```

Comment:
Check highgarden config, I see a few. Also with the cherami module, there is deploymentCluster (I originally had a yaml mapping for it but removed based on your code review comment)